File: 1768-merge-strings-alternately/1768-merge-strings-alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        res = ""
        for i in range(len(word1)):
            res+=word1[i]
            for j in range(len(word2)):
                res += word2[j]
                word2 = word2[j+1:len(word2)]
                break
        if len(word2)>0:
            res+=word2
        return res

File: 1768-merge-strings-alternately/1768-merge-strings-alternately.cpp
class Solution {
public:
    string mergeAlternately(string word1, string word2) {

        int size1 = word1.size();
        int size2 = word2.size();

        int minSize = min(size1, size2);
        int maxSize = max(size1, size2);
        string res = "";

        cout << minSize << endl;

        for (int i = 0; i < minSize; i++){
            res += word1[i];
            res += word2[i];
        }

        for (int i = minSize; i < maxSize; i++){
            if (size1 > minSize)
                res += word1[i];
            if (size2 > minSize)
                res += word2[i];
        }
        
        return res;
    }
};